### ISSUE TITLE
Dashlane makes Facebook unresponsive

### ISSUE THREAD

**Issue by darkgalaxy4** [brave/browser-laptop#8533]
- Did you search for similar issues before submitting this one? Yes

- Describe the issue you encountered: If I use Dashlane to login to Facebook the pointer won't . If I choose LastPass as password manager the problem disappears.

- Platform (Win7, 8, 10? macOS? Linux distro?): Win10x64

- Brave Version (revision SHA):15.0

- Steps to reproduce:
    1. Automatically login to Facebook with Dashlane
    2.Hover with the mouse over any link in the page
    3. Try to open the link
    
- Actual result: I can't open any link when the pointer is hovering 

- Expected result: The links should open

- Will the steps above reproduce in a fresh profile? If not what other info can be added? Yes

- Is this an issue in the currently released version? Yes

- Can this issue be consistently reproduced? Yes

- Extra QA steps:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:

**Comment by Liunkae** [comment]
This problem only happens on facebook.com?

**Comment by darkgalaxy4** [comment]
@Liunkae Yes, other websites behave normally after signing in. The issues didn't show up until the current release.

**Comment by jvventura** [comment]
I have this issue. Except I cannot even login. The login page becomes unresponsive immediately when I interact with the input field.

Here is an error in the console:
<img width="800" alt="screen shot 2017-07-19 at 7 08 38 pm" src="https://user-images.githubusercontent.com/8594182/28393618-b48f4ebc-6cb5-11e7-9400-c31b3f2ad2f9.png">

It looks like the extension is causing a security violation when it loads the iFrame.
